.github/CODEOWNERS

```diff
@@ -73,7 +73,7 @@
 # Does not have owner
 /asset/                                @GoogleCloudPlatform/go-samples-reviewers
 /auth/                                 @GoogleCloudPlatform/go-samples-reviewers
-/dlp/                                  @GoogleCloudPlatform/go-samples-reviewers
+/dlp/                                  @GoogleCloudPlatform/googleapis-dlp @GoogleCloudPlatform/go-samples-reviewers
```

Comment:
issue: it does not look like the googleapis-dlp group is part of the GoogleCloudPlatform organization. Please fix.

Reply:
nit: please keep @GoogleCloudPlatform/go-samples-reviewers as the first entry on the line.

Reply:
@telpirion Thanks for your review.
googleapis-dlp is being used for other doc samples repos.
I am replicating https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/2133/files done by @Sita04 recently. Kindly check if the error message included in description is relevant.

Question reg keeping  @GoogleCloudPlatform/go-samples-reviewers first: does that influence the order in which review is assigned?

Reply:
@telpirion [googleapi-dlp](https://github.com/orgs/GoogleCloudPlatform/teams/googleapis-dlp) exists. Should it be added to the repository permission to make the CODEOWNERS file valid?